src/test/kotlin/com/terraformation/backend/email/EmailNotificationServiceTest.kt

```diff
@@ -1348,6 +1484,25 @@ internal class EmailNotificationServiceTest {
     }
   }
 
+  private fun assertNoMessageSent() {
+    if (sentMessages.isEmpty()) {
+      return
+    } else {
+      fail<Any> {
```

Comment:
Is the `<Any>` needed here? I would have thought type inference would figure out this is a string.

Reply:
It still needed a `<String>` to compile. But I have updated it.